_maps/RandomZLevels/away_mission/snowdin.dmm

```diff
@@ -6281,9 +6281,8 @@
 /obj/effect/turf_decal/tile/yellow{
 	dir = 8
 	},
-/turf/open/floor/plasteel{
```

Comment:
These are skyrat changes that should be marked as such

Reply:
bruh